perception/image_projection_based_fusion/src/segmentation_pointcloud_fusion/node.cpp

```diff
@@ -85,29 +85,44 @@ void SegmentPointCloudFusionNode::fuseOnSingleImage(
   PointCloud2 transformed_cloud;
   tf2::doTransform(input_pointcloud_msg, transformed_cloud, transform_stamped);
 
-  PointCloud output_cloud;
-
-  for (sensor_msgs::PointCloud2ConstIterator<float> iter_x(transformed_cloud, "x"),
-       iter_y(transformed_cloud, "y"), iter_z(transformed_cloud, "z"),
-       iter_orig_x(input_pointcloud_msg, "x"), iter_orig_y(input_pointcloud_msg, "y"),
-       iter_orig_z(input_pointcloud_msg, "z");
-       iter_x != iter_x.end();
-       ++iter_x, ++iter_y, ++iter_z, ++iter_orig_x, ++iter_orig_y, ++iter_orig_z) {
+  int point_step = input_pointcloud_msg.point_step;
+  int x_offset = input_pointcloud_msg.fields[pcl::getFieldIndex(input_pointcloud_msg, "x")].offset;
+  int y_offset = input_pointcloud_msg.fields[pcl::getFieldIndex(input_pointcloud_msg, "y")].offset;
+  int z_offset = input_pointcloud_msg.fields[pcl::getFieldIndex(input_pointcloud_msg, "z")].offset;
+  size_t output_pointcloud_size = 0;
+  output_cloud.data.clear();
+  output_cloud.data.resize(input_pointcloud_msg.data.size());
+  output_cloud.fields = input_pointcloud_msg.fields;
+  output_cloud.header = input_pointcloud_msg.header;
+  output_cloud.height = input_pointcloud_msg.height;
+  output_cloud.point_step = input_pointcloud_msg.point_step;
+  std::cout << "point_step: " << transformed_cloud.point_step << std::endl;
```

Comment:
Is this addition intended? (Or just a debug message which you've forgot to remove?)

Reply:
@kminoda Thanks for comment, It was my typo. I fixed https://github.com/autowarefoundation/autoware.universe/pull/7224/commits/b44464fa858a27542ec862b3aa5d798395db3625